### PR TITLE
Add non recommended card view

### PR DESCRIPTION
**Pull request by davidfranquet**
[FARM-97]

#### 📲 What

Added non-recommended missing widget

#### 🤔 Why
		
Was missing
		
#### ✅ Acceptance criteria

- [ ] Design Review for UI with BA completed. 
- [ ] Manually tested and verified on Android device/emulator (min. Lollipop 5.1)
- [ ] Showcase video / automation test
- [ ] Passing all tests
- [ ] Rebased/merged with latest changes from development and re-tested
- [ ] Removed unsed comments. TODOs must have JIRA for future work.

#### Coding Standards Checklist
- [ ] unit tests 80% coverage on testable code (functions, methods, class)
- [ ] (Architecture) redux state, reducers, actions, middleware defined under lib/redux
- [ ] ui elements defined under lib/ui. 
- [ ] Strings ready for localisation (e.g. defined in lib/utils/strings.dart)
- [ ] Assets (images/icons path) defined in lib/utils/assets.dart
- [ ] Colors defined in lib/utils/colors.dart
- [ ] Dimensions (ie margins padding) defined in lib/utils/dimens.dart
- [ ] TextStyles defined in lib/utils/styles.dart

Recommended Style Guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

#### 🕵️‍♂️ How to test

Notes for QA

#### Reviewers

@farmsmart/amido @farmsmart/wamf

### CODE REVIEW

File: lib/ui/common/recommendation_compact_card/recommendation_compact_card.dart [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+    double overlayIconWidth,
+    String overlayIcon,
+  }) {
+    return RecommendationCompactCardStyle(
+      titleTextStyle: titleTextStyle ?? this.titleTextStyle,
+      subtitleTextStyle: subtitleTextStyle ?? this.subtitleTextStyle,
+      descriptionTextStyle: descriptionTextStyle ?? this.descriptionTextStyle,
+      leftActionButtonStyle:
+          leftActionButtonStyle ?? this.leftActionButtonStyle,
+      rightActionButtonStyle:
```
Comment (LeeMatthewHiggins) [inline lib/ui/common/recommendation_compact_card/recommendation_compact_card.dart:77]: I think the buttons need 2 styles. one normal and one disabled. The VM should then have a bool for if the button is disabled or not. (the same thing for the hero recommended card) The VM can't change styles. but the widget (view layer) can based on a VM var.

**Review by LeeMatthewHiggins — CHANGES_REQUESTED** [review]

File: lib/ui/common/recommendation_compact_card/recommendation_compact_card.dart (hunk 1 above)
Comment (davidfranquet) [reply lib/ui/common/recommendation_compact_card/recommendation_compact_card.dart:77]: @LeeMatthewHiggins DONE, you can check the RoundedButtonStateful, you can pass 2 styles for each button and it has a bool to display the first or the second (as talked)

File: lib/ui/common/recommendation_card/recommendation_card.dart [hunk 2]
```diff
@@ -269,11 +266,13 @@ class RecommendationCard extends StatelessWidget {
       child: Row(
         children: <Widget>[
           Expanded(
-            child: RoundedButton(
+            child: RoundedButtonStateFul(
```
Comment (LeeMatthewHiggins) [inline lib/ui/common/recommendation_card/recommendation_card.dart]: RoundedButtonStateful.    typo..

**Review by LeeMatthewHiggins — COMMENTED** [review]
just a minor typo

File: lib/ui/common/recommendation_card/recommendation_card.dart (hunk 2 above)
Comment (davidfranquet) [reply lib/ui/common/recommendation_card/recommendation_card.dart]: 👍

Comment (davidfranquet) [reply lib/ui/common/recommendation_card/recommendation_card.dart]: Fixed

**Review by LeeMatthewHiggins — APPROVED** [review]

**Pull request merged**